rocAL/rocAL/CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ if(${BUILD_RALI})
     else()
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_OPENCV=0)
     endif()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fopenmp -msse4.2 -mavx2 -Wall  -fPIC -pg -pthread -std=c++14 -Wno-deprecated-declarations")
```

Comment:
Why are we disabling "Wall"? It is better to leave this on to generate better code